src/v/kafka/server/group_metadata_migration.cc

```diff
@@ -687,11 +687,15 @@ ss::future<> group_metadata_migration::start(ss::abort_source& as) {
           mlog.info,
           "kafka_internal/group topic does not exists, activating "
           "consumer_offsets feature");
-        try {
-            co_await activate_feature(as);
-        } catch (const ss::abort_requested_exception&) {
-            // ignore abort requested exception, we are shutting down
+
+        _sub = as.subscribe([this]() noexcept { _as.request_abort(); });
+        if (!_sub) {
+            _as.request_abort();
         }
+
+        ssx::spawn_with_gate(_background_gate, [this]() -> ss::future<> {
+            return activate_feature(_as);
```

Comment:
I think we should add here retries in case if `kafka_internal::group` topic was created here while waiting for feature to be active. Can you add different implementation here for `activate_feature` that will also check in the loop if `kafka_internal::group` is present ?. If the topic was created it should execute `do_apply` code path